ci-opal.yml

```diff
@@ -134,7 +135,9 @@ provisioners:
   - /tmp/chef-stuff/travis-cookbooks/cookbooks
   - /tmp/chef-stuff/travis-cookbooks/community-cookbooks
   <% end %>
-  json: {}
+  json:
+    travis_packer_build:
+      github_token: "{{ user `github_token` }}"
```

Comment:
Fair warning that this token will be published as part of the image metadata, so it should be a "no scope" token strictly intended for authenticated API usage.